bayesian/api/user_api.py

```diff
@@ -49,6 +49,11 @@ def create_or_update_user():
 
     user_id = content.get('user_id')
 
+    # Needs to be reverted after rds upgrade
+    if 1 == 1:
```

Comment:
why 1 == 1?

Reply:
I didnt wanna remove other portion of the code just for this.
so instead of returning i thought of adding a condition whihc is always true.

Reply:
Open to better methodologies.